packages/cloudflare/src/cli/build/patches/ast/patch-vercel-og-library.ts

```diff
@@ -18,15 +18,16 @@ type TraceInfo = { version: number; files: string[] };
 export function patchVercelOgLibrary(buildOpts: BuildOptions) {
   const { appBuildOutputPath, outputDir } = buildOpts;
 
-  const packagePath = path.join(outputDir, "server-functions/default", getPackagePath(buildOpts));
+  const functionsPath = path.join(outputDir, "server-functions/default");
+  const packagePath = path.join(functionsPath, getPackagePath(buildOpts));
 
   for (const traceInfoPath of globSync(path.join(appBuildOutputPath, ".next/server/**/*.nft.json"))) {
     const traceInfo: TraceInfo = JSON.parse(readFileSync(traceInfoPath, { encoding: "utf8" }));
     const tracedNodePath = traceInfo.files.find((p) => p.endsWith("@vercel/og/index.node.js"));
 
     if (!tracedNodePath) continue;
 
-    const outputDir = path.join(packagePath, "node_modules/next/dist/compiled/@vercel/og");
+    const outputDir = getOutputDir({ functionsPath, packagePath });
```

Comment:
question: could this be derived from the location of the trace file?

Reply:
not from the nft files from what i saw unless i missed something. the existence check could be changed to be checking if the normal package directory's node_modules contains it (apps/web/node_modules), but i stuck to doing the existence check on our output directory instead of their project's directory.

Reply:
Thanks for the explanation.

_(If we drop copying the traced files as mentioned yesterday, we wouldn't even have to copy...)_